### ISSUE TITLE
No requests listed at first start

### ISSUE THREAD

**Issue by kvlahromei** [CodeForEindhoven/open311-android#62]
(possible dublicat of #38)

When you start the app for the first time (and didn't enter any requests yet) you see only the hint for non-existing requests. Internally the app fires an confusing API call
`/citysdk/requests.json?jurisdiction_id=yourid&service_request_id=9999`
(If this id is non-existing, the server returns an empty list like Rostock, but others seem to be more tolerant as Bonn for example)

Maybe it's easier to list all of the requests, or at least e.g. the 25 recent ones?

**Comment by milovanderlinden** [comment]
Agreed and indeed slightly duplicate to #38, but as you suggest an elegant
solution, I will close #38 in favor of this issue.

**Comment by kvlahromei** [comment]
Hi @milovanderlinden  do you have any fixes in the next version? We are curious, to checkout the app with our live endpoint 😄
